plugins/Files/js/components/app.js

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react'
+import FileBrowser from '../containers/filebrowser.js'
+import AllowanceDialog from '../containers/allowancedialog.js'
+
+const FilesApp = ({unlocked, showAllowanceDialog}) => {
+	let fileBrowserContent
+	if (!unlocked) {
+		fileBrowserContent = (
```

Comment:
agreed, I changed this to be a single expression and also added styling ("You must unlock..." was unstyled previously.)